.github/release-drafter.yml

```diff
@@ -0,0 +1,45 @@
+name-template: 'v$NEXT_MINOR_VERSION'
+tag-template: 'v$NEXT_MINOR_VERSION'
+categories:
+  - title: '🚀 Features'
```

Comment:
shall we use the names and follow the style for those names from the release notes https://docs.nginx.com/nginx-ingress-controller/releases/ ?

we have more granular feature and improvements section, but that is something we can add and edit before the release manually

Reply:
I feel like the way it is now, it's a little bit flat, hard to find the different sections. Also, do we really need that granularity?